### ISSUE TITLE
Any Refraction IOR value except 1.0 creates black artifacts

### ISSUE THREAD

**Issue by jrz371** [GPUOpen-LibrariesAndSDKs/RadeonProRenderSDK#26]
Tested with RadeonProRenderSDK 2.02.2, also reproducible in the Blender addon 
![image](https://user-images.githubusercontent.com/16514346/114702655-05b88500-9d2d-11eb-88f7-d57697bb9ff1.png)

Using any refraction IOR value except 1.0 creates these black rendering artifacts.

**Comment by jrz371** [comment]
Here's an example made in Blender 2.91
[IOR bug Blender.zip](https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderSDK/files/6315490/IOR.bug.Blender.zip)

**Comment by SimonDanisch** [comment]
Did you try setting RPR_CONTEXT_MAX_RECURSION to e.g. 10?

**Comment by jrz371** [comment]
Yes, we use 8 as a default and even at 10 there is no discernable difference.
